tasks/Functions-client/deployAutoClient.js

```diff
@@ -77,7 +71,12 @@ task("functions-deploy-auto-client", "Deploys the AutomatedFunctionsConsumer con
     console.log(`\nAdded consumer contract ${consumerAddress} in Tx: ${addConsumerTx.transactionHash}`)
 
     const verifyContract = taskArgs.verify
-    if (verifyContract && !!networks[network.name].verifyApiKey && networks[network.name].verifyApiKey !== "UNSET") {
+    if (
+      network.name !== "localFunctionsTestnet" &&
```

Comment:
This logic needs to be separated out otherwise the else logic on line 50 will run for the local testnet and tell the user that they have not set the API keys (even if they have)

The situation where `network.name == "localFunctionsTestnet" ` should just silently skip verification, so before the assignment of taskargs.verify we could just do
```
if (network.name === "localFunctionsTestnet) {
  return 
}
```